e2e/test/scenarios/models/create.cy.spec.js

```diff
@@ -47,7 +47,7 @@ describe("scenarios > models > create", () => {
     H.visitCollection(THIRD_COLLECTION_ID);
 
     navigateToNewModelPage();
-    H.NativeEditor.focus().type("select * from ORDERS");
+    H.NativeEditor.focus().type("select 1");
```

Comment:
Not really related to this PR but couldn't help it.
What we `SELECT` was completely irrelevant for this test. It only needed a valid SQL query/result.